pkg/controller/direct/securesourcemanager/mapper.generated.go

```diff
@@ -49,10 +49,11 @@ func Instance_PrivateConfig_FromProto(mapCtx *direct.MapContext, in *pb.Instance
 	}
 	out := &krm.Instance_PrivateConfig{}
 	out.IsPrivate = direct.LazyPtr(in.GetIsPrivate())
-	out.CaPool = direct.LazyPtr(in.GetCaPool())
+	if in.GetCaPool() != "" {
+		out.CaPoolRef = &refs.PrivateCACAPoolRef{External: in.GetCaPool()}
```

Comment:
We normally do not call resolvers in mappers. References should be resolved in controller before we convert KRM to/from proto. 

To avoid confusion I suggest moving these out of mapper.generated.go, for example: https://github.com/GoogleCloudPlatform/k8s-config-connector/blob/95b3225e0040b41db93bb05d26bd4d6007dab44b/pkg/controller/direct/compute/forwardingrule/mapper.go#L55

Reply:
moved to mapper.go